specifyweb/frontend/js_src/lib/components/Atoms/Form.tsx

```diff
@@ -268,7 +268,7 @@ export const Select = wrap<
 >(
   'Select',
   'select',
-  `${className.notTouchedInput} w-full pr-5 bg-right cursor-pointer`,
+  `${className.notTouchedInput} w-full pr-5 bg-right cursor-pointer min-w-[theme(spacing.40)]`,
```

Comment:
Just to double-check, you meant to set min-width for just one type of field? Not for all of them?